packages/libs/src/services/solana/transfer.ts

```diff
@@ -292,7 +292,7 @@ export const createTransferInstructions = async ({
   const transferDataInstr = Uint8Array.of(1, ...ethAddressArr)
   const transferInstruction = new TransactionInstruction({
     keys: accounts,
-    programId: claimableTokenProgramKey.toString() as unknown as PublicKey,
+    programId: new PublicKey(claimableTokenProgramKey.toString()),
```

Comment:
can we just use PublicKey instead here? keep things consistent? i remember something broke with v0 when passing in a string programId.

Reply:
😆 yes you're correct, `claimableTokenProgramKey` already being a `PublicKey` makes this so redundant!